### PR TITLE
Compile expense report

### PR DESCRIPTION
**Pull request by TheLillyleaf**
Enables creating an expense report with included test to see report

### CODE REVIEW

File: Purchase/index.ts [hunk 1]
```diff
@@ -133,6 +134,88 @@ export namespace Purchase {
...
+		const headerSize = Math.round(fontSize * 1.33)
+		const xMargin = 30
+		const yMargin = 4 * fontSize
+		const cellWidth = 230
+		const lineHeight = 15
+		const lineThickness = 1
+		const lineMargin = 1
+		const headers = ["Buyer", "Purpose", "Amount"]
+		// const receiptsPerIndexPage = (height - 2 * yMargin - fontSize / 2) / lineHeight
+		// const costCenterStartPage: Record<string, number> = {}
```
Comment (TheLillyleaf) [inline Purchase/index.ts:161]: remove

File: Purchase/index.ts [hunk 2]
```diff
@@ -133,6 +134,88 @@ export namespace Purchase {
...
+		dateRange: isoly.DateRange
+	): Promise<Uint8Array> {
+		let result: Uint8Array | undefined = undefined
+		const pdfDocument = await PDFLib.PDFDocument.create()
+		const font = await pdfDocument.embedFont(PDFLib.StandardFonts.Courier)
+		pdfDocument.setAuthor("Issuefab AB")
+		pdfDocument.setCreationDate(new Date())
+		const width = PDFLib.PageSizes.A4[0]
+		const height = PDFLib.PageSizes.A4[1]
+
```
Comment (TheLillyleaf) [inline Purchase/index.ts:150]: remove

**Review by TheLillyleaf — COMMENTED** [review]
self check

**Review by TheLillyleaf — COMMENTED** [review]
self check

**Review by EliasEriksson — APPROVED** [review]

**Review by EliasEriksson — APPROVED** [review]

**Pull request merged**